python_bindings/requirements.txt

```diff
@@ -11,5 +11,6 @@ pillow
 
 # Keep versions of these requirements equal to the versions in Ubuntu 20.04 LTS
 # because newer versions fix bugs we work around.
+# TODO: what bugs? how are we working around them? is this still valid?
```

Comment:
The bugs are/were with the pybind11 CMake package (before 2.5) not using the Python3 `find_package` stuff. This comment is stale since we've (apparently) decided to not support the Ubuntu pybind11. You can go ahead and remove that comment.

Reply:
PTAL

Reply:
:+1: